src/disco/containers/DiscoPane.js

```diff
@@ -72,7 +72,9 @@ export class DiscoPane extends React.Component {
         </header>
         {results.map((item, i) => <Addon {...camelCaseProps(item)} key={i} />)}
         <div className="amo-link">
-          <a href="https://addons.mozilla.org/">{i18n.gettext('See more add-ons!')}</a>
+          <a href="https://addons.mozilla.org/" target="_blank">
```

Comment:
I was thinking we might want to log that they came from the disco pane. Maybe that logging is better off in here though?